### PR TITLE
Trackr setup documentation - initial; Closes Issue #18

### PR DESCRIPTION
**Pull request by anielse2**
Initial addition of Trackr setup doc and updated event source image.

### CODE REVIEW

File: docs/Trackr_Setup.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Trackr
```
Comment (amyschoen) [inline docs/Trackr_Setup.md]: Looks mostly good.  I suggest starting with "#Trackr Initial Setup" to make it a header.  It would look a little nicer.

**Comment by anielse2** [comment]
Requested change made.

**Comment by amyschoen** [comment]
Looks like I steered you wrong.  😊 You need a space between the # and the title text.

**Comment by anielse2** [comment]
Fixed now. :)

**Comment by amyschoen** [comment]
Looks good.  I'll merge everything in.

**Pull request merged**